test/unit/alignment/multiple/align_multiple_test.cpp

```diff
@@ -0,0 +1,145 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <string_view>
+#include <vector>
+
+#include <seqan3/alignment/configuration/align_config_gap.hpp>
+#include <seqan3/alignment/configuration/align_config_scoring_scheme.hpp>
+#include <seqan3/alignment/scoring/aminoacid_scoring_scheme.hpp>
+#include <seqan3/alignment/multiple/align_multiple.hpp>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
+#include <seqan3/alphabet/nucleotide/dna15.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/core/detail/debug_stream_alphabet.hpp>
+#include <seqan3/range/views/to_char.hpp>
+#include <seqan3/range/views/to.hpp>
+#include <seqan3/range/views/deep.hpp>
+#include <seqan3/test/expect_range_eq.hpp>
+
+using seqan3::operator""_dna15;
+using seqan3::operator""_dna4;
+using seqan3::operator""_dna5;
+using seqan3::operator""_rna5;
+using seqan3::operator""_rna4;
+using seqan3::operator""_aa27;
+
+TEST(align_multiple_test, the_first_dna4_test)
+{
+    std::vector<seqan3::dna4_vector> input{"AAAACCCGGG"_dna4, "AACCCGGG"_dna4, "AAAACGGG"_dna4};
+    std::vector<std::string> output{"AAAACCCGGG", "--AACCCGGG", "--AAAACGGG"};
```

Comment:
```suggestion
    std::vector<seqan3::dna4_vector> const input{"AAAACCCGGG"_dna4, "AACCCGGG"_dna4, "AAAACGGG"_dna4};
    std::vector<std::string> const output{"AAAACCCGGG", "--AACCCGGG", "--AAAACGGG"};
```

The input should not be changed by the algorithm, right?

Also the other test cases